bin/perlbug

```diff
@@ -2,7 +2,7 @@
     eval 'exec /home/git/binary-com/perl/bin/perl -S $0 ${1+"$@"}'
```

Comment:
TO reviewer:
The only place I changed is in the `rebuild.sh` . so please review that file directly.